notebooks/demo/nx_cugraph_demo.ipynb

```diff
@@ -0,0 +1,672 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# `nx-cugraph`: a NetworkX backend that provides GPU acceleration with RAPIDS cuGraph"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook will demonstrate the `nx-cugraph` NetworkX backend using the NetworkX betweenness_centrality algorithm.\n",
+    "\n",
+    "## Background\n",
+    "Networkx version 3.0 introduced a dispatching mechanism that allows users to configure NetworkX to dispatch various algorithms to third-party backends. Backends can provide different implementations of graph algorithms, allowing users to take advantage of capabilities not available in NetworkX. `nx-cugraph` is a NetworkX backend provided by the [RAPIDS](https://rapids.ai) cuGraph project that adds GPU acceleration to greatly improve performance.\n",
+    "\n",
+    "## System Requirements\n",
+    "Using `nx-cugraph` with this notebook requires the following: \n",
+    "- NVIDIA GPU, Pascal architecture or later\n",
+    "- CUDA 11.2, 11.4, 11.5, 11.8, or 12.0\n",
+    "- Python versions 3.9, 3.10, or 3.11\n",
+    "- NetworkX >= version 3.2\n",
+    "  - _NetworkX 3.0 supports dispatching and is compatible with `nx-cugraph`, but this notebook will demostrate features added in 3.2_\n",
```

Comment:
```suggestion
    "  - _NetworkX 3.0 supports dispatching and is compatible with `nx-cugraph`, but this notebook will demonstrate features added in 3.2_\n",
```

Reply:
Thanks for finding this and the other typos below. I think I should also be using `networkx` and NetworkX more consistently in the latest commits.